src/pages/saksbehandling/avsluttBehandling/avsluttBehandlingBunnknapper/AvsluttBehandlingBunnknapper.tsx

```diff
@@ -0,0 +1,34 @@
+import * as RemoteData from '@devexperts/remote-data-ts';
+import { Back } from '@navikt/ds-icons';
+import { Button, Loader } from '@navikt/ds-react';
+import * as React from 'react';
+
+import LinkAsButton from '~components/linkAsButton/LinkAsButton';
+import { useI18n } from '~lib/i18n';
+import * as Routes from '~lib/routes';
+
+import messages from '../avsluttBehandling-nb';
+
+import styles from './avsluttBehandlingBunnknapper.module.less';
+
+const AvsluttBehandlingBunnknapper = (props: {
+    submitButtonText: string;
+    submitStatus: RemoteData.RemoteData<unknown, unknown>;
```

Comment:
Kanskje det er bedre å kalle denne `prop`-en for `isLoading` eller `isPending` og la det være en boolean? Så blir komponenten litt dummere, og vi slipper `unknown` types :D